### PR TITLE
Make progress bar reflect number of processed files + process file by file in series

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by sabbelasichon** [comment]
@TomasVotruba We have to deal with a chicken egg problem here, that´s the reason why most of the workflows failed.  But that´s how i solved it basically in one of my historical pull requests. Looks great to me.

**Comment by TomasVotruba** [comment]
Let's ship it then :)

File: .github/workflows/build_scoped_rector.yaml [hunk 1]
```diff
@@ -37,8 +37,7 @@ jobs:
 
             # early downgrade individual functions and files of symfony Attribute classes
             -   run: bin/rector process src/functions -c build/config/config-downgrade.php --ansi
-            -   run: bin/rector process vendor/symfony/dependency-injection/Attribute/Autoconfigure.php -c build/config/config-downgrade.php --ansi
-            -   run: bin/rector process vendor/symfony/dependency-injection/Attribute/AutoconfigureTag.php -c build/config/config-downgrade.php --ansi
+            -   run: bin/rector process vendor/symfony/dependency-injection/Attribute -c build/config/config-downgrade.php --ansi
```
Comment (samsonasik) [inline .github/workflows/build_scoped_rector.yaml:40]: @TomasVotruba previuos one by one run on `Autoconfigure.php` and `AutoconfigureTag.php` is on purpose as not works to touch both files.

Comment (samsonasik) [reply .github/workflows/build_scoped_rector.yaml:40]: Ok, it seems reverted at latest main 

https://github.com/rectorphp/rector-src/blob/96dbc6956249779658a30c7e1a05a5ec396e94d7/.github/workflows/build_scoped_rector.yaml#L40-L41

Comment (TomasVotruba) [reply .github/workflows/build_scoped_rector.yaml:40]: I tried it because this should work either way. What is the desired change of those 2 files?

Comment (samsonasik) [reply .github/workflows/build_scoped_rector.yaml:40]: that should remove the Attribute, and by use the directory, it skipped them.

Comment (TomasVotruba) [reply .github/workflows/build_scoped_rector.yaml:40]: That's weird, I'll look into it.

Also, the attribute is interpreted as a comment in older PHP, so this change is probably not needed. The `#[Attribute]` can be left there as a comment
https://3v4l.org/7k941

Do you recall what real problems absence of these 2 run lines caused?

Comment (samsonasik) [reply .github/workflows/build_scoped_rector.yaml:40]: previously, it error:

```bash
Uncaught TypeError: PhpParser\Internal\TokenStream::getIndentationBefore(): Return value must be of type int, null returned
```
see https://github.com/rectorphp/rector-src/pull/133#issuecomment-851972838

Looking at latest main, it seems the : 

```php
new DowngradeAttributeToAnnotation(
                     'Attribute',
                     'Attribute'
),
```

it removed. See https://github.com/rectorphp/rector-src/commit/02f9524e3414e9f65c38611dfc38ccefb5257695#diff-c195c143f1dc6e27a371a830fc13d5d65d089c21c89483c9766d650b7ba46484R21

Comment (samsonasik) [reply .github/workflows/build_scoped_rector.yaml:40]: The real problem seems as the DowngradeAnnotation not run, the `DowngradeNamedArgumentRector` not executed, that make fatal error.

Comment (samsonasik) [reply .github/workflows/build_scoped_rector.yaml:40]: I created PR for by directory at https://github.com/rectorphp/rector-src/pull/367

**Pull request merged**